pulsar-io/debezium/core/src/main/java/org/apache/pulsar/io/debezium/DebeziumSource.java

```diff
@@ -60,11 +62,24 @@ public static String topicNamespace(SourceContext sourceContext) {
                 + (StringUtils.isEmpty(namespace) ? TopicName.DEFAULT_NAMESPACE : namespace);
     }
 
+    public static void tryLoadingConfigSecret(String secretName, Map<String, Object> config, SourceContext context) {
+        try {
+            String secret = context.getSecret(secretName);
+            if (secret != null) {
+                config.put(secretName, secret);
```

Comment:
it's better to output that we've just overwritten the user config. This would also helps troubleshooting

something like

log.info("Config key {} set from secret", secretName)

Reply:
Good point, added!